src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return 40 * days - 50;
+  } else if (days >= 3) {
+    return 40 * days - 20;
+  } else {
+    return 40 * days;
+  }
```

Comment:
```suggestion
  }
return 40 * days;
```
you could omit last else